app/client/src/types.ts

```diff
@@ -45,6 +41,21 @@ export type AnalysisStatus = {
     [key in AnalysisType]: string | null
 }
 
-export interface ClusterInfo {
-    data: Dict<Dict<string>>
+export type ClusterInfo = Dict<Dict<string>>
+
+export interface BeebopError {
+    error: string,
+    detail?: string
+}
+
+export interface ResponseFailure {
+    status: 'failure';
+    data: null;
+    errors: BeebopError[];
+}
+
+export interface ResponseSuccess<T=unknown> {
+    status: 'success';
+    data: T;
+    errors: null;
```

Comment:
What was the reason for changing this from the original?:
```
export interface ResponseSuccess {
    status: "success";
    data: unknown;
    errors: null;
}
```

Reply:
I once wanted to use something like ResponseSuccess<ClusterInfo>, but have discarded this. Will change this back to previous version